src/sentry/static/sentry/app/routes.jsx

```diff
@@ -188,31 +188,32 @@ const orgSettingsRoutes = [
     component={errorHandler(OrganizationGeneralSettingsView)}
   />,
 
-  <Route
-    key="team-details"
-    name="Teams"
-    path="teams/:teamId/"
-    component={errorHandler(TeamDetails)}
-  >
-    <IndexRedirect to="settings/" />
-    <Route path="settings/" name="Settings" component={errorHandler(TeamSettings)} />
-    <Route path="members/" name="Members" component={errorHandler(TeamMembers)} />
-  </Route>,
+  <Route key="teams" name="Project & Teams" path="teams/">
+    <IndexRedirect to="your-teams" />
+    <Route
+      path="all-teams/"
+      name="All Teams"
+      allTeams
```

Comment:
what is this doing? does Route accept this

Reply:
Yeah route just passes all props to the route object, probably don't need this anymore, but it was added for the teams/all teams tab